scripts/build.sh

```diff
@@ -36,11 +36,10 @@ if [ "${TRAVIS_BRANCH}" == "master" ] && [ "${TRAVIS_PULL_REQUEST}" == "false" ]
   set +e
   git add .
   set -e
-  git status
```

Comment:
I think it's more for debugging purposes. Removing it won't hurt anything but in the future we might need to add it back if we're (for whatever reason) not having our docs built properly. Leave it removed, we should be ok.